appveyor.yml

```diff
@@ -78,16 +78,17 @@ test_script:
 #      artifacts configuration    #
 #---------------------------------#
 
-#artifacts:
```

Comment:
Did you mean to remove this or just uncomment it?

Reply:
I have a fix for this, I noticed after the initial push